### DISCUSSION TITLE
Is it possible to get this extension on VS Codium

### DISCUSSION THREAD

**Discussion by AmandeepKataria** [mark-wiemer/ahkpp#43] [Q&A] [answered]
I've recently downloaded VS Codium with the hopes that all packages that are available on VS code are available on this also. Was wondering if this extension is also available on there.

**Comment by FuPeiJiang** [comment]
@AmandeepKataria please add refs:
publish to [open-vsx.org](https://open-vsx.org/) in addition to the VS Code Marketplace. The publishing process is documented in the [Open VSX Wiki](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions).

[Extensions + Marketplace](https://github.com/VSCodium/vscodium/blob/master/DOCS.md#extensions-marketplace)

[Why Does This Exist](https://github.com/VSCodium/vscodium#why)

...
But then I read https://www.reddit.com/r/programming/comments/bc23z4/vscodium_100_open_source_version_of_microsoft_vs#t1_ekne2o6
And
https://www.reddit.com/r/programming/comments/bc23z4/vscodium_100_open_source_version_of_microsoft_vs#t1_eknjsie

___
I like the microsoft-vscode logo better

can you convince me to use VS Codium ?

**Comment by mark-wiemer** [comment, marked as answer]
Thanks for your suggestion. True OSS is always great, and VS Codium seems like a useful project. That said, I won't outright say no, but it's definitely low-priority for now.

I had never heard of Codium, and I kind of understand why it exists (anti-telemetry, true OSS), but I have a hard time believing MSFT's mandatory telemetry policy is overly intrusive (yes, as an MSFT employee, I know I'm biased). The `.vsix` files for each build are available for download in the [Releases](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/releases) of this repo, you can easily install from there. In both Code and Codium you can move the `.vsix` file into an open folder, then select "Install Extension VSIX" from the context menu (right-click menu):

![](https://user-images.githubusercontent.com/7833360/107309982-561b1a00-6a40-11eb-9ebd-5c895bfd9653.png)

I know it's a bit of a pain to manually install updates, but I am focusing on developer tooling and automation so that even the current publishing process becomes simpler. Once that's automated, it shouldn't be so hard to automate publication to VS Codium.
